AppController/djinn.rb

```diff
@@ -3609,12 +3609,14 @@ def start_api_services()
       threads << Thread.new {
         Djinn.log_info("Starting database services.")
         clear_datastore = @options['clear_datastore'].downcase == "true"
-        replication = Integer(@options['replication'])
+        db_nodes = @nodes.count{|node| node.is_db_master? or node.is_db_slave?}
+        needed_nodes = needed_for_quorum(db_nodes,
```

Comment:
Would this be a problem in migration cases where the replication factor increases or decreases?